frontend/src/components/modal/mobile/photo/PhotosGrid.jsx

```diff
@@ -8,27 +8,27 @@ import GridSmallItemRight from '../styled-components/GridSmallItemRight';
 import GridImg from '../styled-components/GridImg';
 
 const PhotosGrid = (props) => (
-  <Flex>
-    <PhotoContainer>
-      <PhotoGrid>
-        {props.big && (
-          <GridBigItem>
-            <GridImg src={props.big} alt={'house'} />
-          </GridBigItem>
-        )}
-        {props.left && (
-          <GridSmallItemLeft>
-            <GridImg src={props.left} alt={'house'} />
-          </GridSmallItemLeft>
-        )}
-        {props.right && (
-          <GridSmallItemRight>
-            <GridImg src={props.right} alt={'house'} />
-          </GridSmallItemRight>
-        )}
-      </PhotoGrid>
-    </PhotoContainer>
-  </Flex>
+    <Flex>
+        <PhotoContainer>
+            <PhotoGrid>
+                {props.big && (
```

Comment:
at first glance im not sure what `big` is other than, its big. if this is a grid layout, i might suggest indicating that with `gridBig` etc.

Reply:
I agree, I could have been more specific here. It's referring to the pattern of the mobile modal view of one big image, and two small images under it that are next to each other (left & right). I am not always too sure how to be descriptive with grid layouts.